Makefile

```diff
@@ -47,6 +47,7 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+.PHONY: all
```

Comment:
Are these changes required for the sdk-operator upgrade? If so, please mention why in the commit message; otherwise, I recommend moving them to a separate PR.

Reply:
Yes they are regarded as part of the OSDK upgrade. See https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.16.0/#add-phony-targets-to-makefile

Reply:
Please add such links to your commit message. This will explain WHY you are making the change.

Reply:
Will add links to this and the rest of the commits